hadoop-hdfs-project/hadoop-hdfs/pom.xml

```diff
@@ -225,6 +225,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>lz4-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Please update LICENSE-binary to include this new dependency.
Also, please manage the dependency version in hadoop-project/pom.xml